web_no_crawler/README.rst

```diff
@@ -0,0 +1,35 @@
+**This file is going to be generated by oca-gen-addon-readme.**
```

Comment:
Generates the README with oca-gen-addon-readme

Reply:
Not entirely clear what you mean but do you want me to run this script and commit its result? Is it not meant to be generated automatically?

Reply:
I normally generate README.rst using this script before submitting something. However, it may be that at the time of MERGE it will be automatically generated (I'm not sure)

Reply:
https://github.com/OCA/maintainer-tools#quality-tools

Reply:
@marcelsavegnago that one will be generated by the ocabot as @StefanRijnhart explained.

Reply:
You're a man of few words I must say.

Anyway, I read on that page `To provide high quality README for our modules we generate them automatically.` And indeed, it looks like the OCA git bot is updating the readme's in the repos. Here is an example of that: https://github.com/OCA/e-commerce/commit/3a60dcc4cf4a639d6044b7c875e0dced5366b14b

Reply:
Thanks for confirming @rvalyi. Can't say I'm very much up to date with all the magic of the OCA ;-)

Reply:
Thanks for the confirmation @rvalyi . I apologize for the confusion @StefanRijnhart :-D

Reply:
No problem. Thanks for the review!